docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -81,20 +81,63 @@ EOF
+* You have defined the shell variables to run the script that runs the upgrade:
 +
 ----
-CEPH_FSID=$(oc get secret ceph-conf-files -o json | jq -r '.data."ceph.conf"' | base64 -d | grep fsid | sed -e 's/fsid = //'
+$ CEPH_FSID=$(oc get secret ceph-conf-files -o json | jq -r '.data."ceph.conf"' | base64 -d | grep fsid | sed -e 's/fsid = //'
 
-alias openstack="oc exec -t openstackclient -- openstack"
-declare -A computes
-export computes=(
-  ["standalone.localdomain"]="192.168.122.100"
+$ alias openstack="oc exec -t openstackclient -- openstack"
+
+$ DEFAULT_CELL_NAME="cell3" <1>
+$ RENAMED_CELLS="cell1 cell2 $DEFAULT_CELL_NAME"
+
+$ declare -A COMPUTES_CELL1
+$ export COMPUTES_CELL1=( <2>
+  ["standalone.localdomain"]="192.168.122.100" <3>
+  # ... <4>
+)
+$ declare -A COMPUTES_CELL2
+$ export COMPUTES_CELL2=(
+  # ...
+)
+$ declare -A COMPUTES_CELL3
+$ export COMPUTES_CELL3=(
+  # ... <5>
+)
+# ...
+
+$ declare -A COMPUTES_API_CELL1
+$ export COMPUTES_API_CELL1=( <2>
+  ["standalone.localdomain"]="172.17.0.100"
   # ...
 )
+# ...
+
+$ NODESETS=""
+$ for CELL in $(echo $RENAMED_CELLS); do
+  ref="COMPUTES_$(echo ${CELL}|tr '[:lower:]' '[:upper:]')"
+  eval names=\${!${ref}[@]}
+  [ -z "$names" ] && continue <6>
+  NODESETS="'openstack-${CELL}', $NODESETS"
+done
+$ NODESETS="[${NODESETS%,*}]"
 ----
 +
-** Replace `["standalone.localdomain"]="192.168.122.100"` with the name and IP address of the {compute_service} node.
+<1> The source cloud `default` cell takes a new `DEFAULT_CELL_NAME` on the destined cloud after adoption.
+In a multi-cell adoption scenario, you may either retain its original name `default`, or create as `cell<X>`, by providing the incremented index of the last cell in the source cloud (which is, by adding a 1 to it).
+<2> For each cell, adjust <["standalone.localdomain"]="x.x.x.x">, and complete `COMPUTES_CELL<X>` and `COMPUTES_API_CELL<X>` data with the names and IP addresses of the {compute_service} nodes connected to the `ctlplane` and `internalapi` networks. Do not specify a real FQDN defined for each network, always use the same host name for each of its connected networks. Providing IP addresses and names of the hosts on remaining networks of the source cloud should be covered the similar way. Or you can adjust the generated files manually as shown below.
+<3> If your deployment has a custom DNS Domain, put it in for FQDN of the nodes. The given values will be used in the dataplane node sets' `spec.nodes.<NODE NAME>.hostName`.
+<4> Assign all {compute_service} nodes from the source cloud `cell1` cell into `COMPUTES_*CELL1`, and so on.
+<5> Assign all {compute_service} nodes from the source cloud `default` cell into `openstack-<X>`,
+where `<X>` is the `DEFAULT_CELL_NAME` environment variable value (here, it equals 'cell3').
+<6> Cells not containing compute nodes will be omitted as no node sets for it should be created.
+
+** A standalone TripleO only creates a default cell, so you should define that instead:
```

Comment:
None of the production deployments are Standalone, AFAIK, so we should either remove those mentions or put them inside a `ifeval::["{build}" != "downstream"]` condition.

Reply:
We have agreed to address the docs review in follow up